plotpal/volumes.py

```diff
@@ -0,0 +1,350 @@
+        # Add and setup colorbar & label
+        norm = matplotlib.colors.Normalize(vmin=vmin, vmax=vmax)
+
+        cb = matplotlib.colorbar.ColorbarBase(cax, cmap=cmap,
+                                norm=norm,
+                                orientation='horizontal')
+        cb.solids.set_rasterized(True)
+        cb.set_ticks((vmin, vmax))
+        cax.tick_params(direction='in', pad=1)
+        cb.set_ticklabels(('{:.2e}'.format(vmin), '{:.2e}'.format(vmax)))
+        cax.xaxis.set_ticks_position('bottom')
+        if self.label is not None:
+            #if self.vector_ind is not None:
+                #cax.text(0.5, 0.5, '{:s}[{}]'.format(self.task, self.vector_ind), transform=cax.transAxes, va='center', ha='center')
+           # else:
+            #    cax.text(0.5, 0.5, '{:s}'.format(self.task), transform=cax.transAxes, va='center', ha='center')
+        #else:
+            cax.text(0.5, 0.5, '{:s}'.format(self.label), transform=cax.transAxes, va='center', ha='center')
+        return cb
+     
+
+    def plot_colormesh(self, ax, cax, dsets, ni, **kwargs):
+        
+        if self.first:
+            x = match_basis(dsets[self.top], self.x_basis)
+            y = match_basis(dsets[self.top], self.y_basis)
+            z = match_basis(dsets[self.left], self.z_basis)
+            self.x = x
+            self.y = y
+            self.z = z
+            self.Lx = x[-1]
+            self.Ly = y[-1]
+            self.Lz = z[-1]
+
+        left_field = np.squeeze(dsets[self.left][ni,:])
+        right_field = np.squeeze(dsets[self.right][ni,:])
+        top_field = np.squeeze(dsets[self.top][ni,:])
+        
+        vector_ind = self.vector_ind
+        if vector_ind is not None:
+            left_field = left_field[vector_ind,:]
+            right_field = right_field[vector_ind,:]
+            top_field = top_field[vector_ind,:]
+        
+        left_field = self._modify_field(left_field)
+        right_field = self._modify_field(right_field)
+        top_field = self._modify_field(top_field)
+        
+        xy_side = construct_surface_dict(self.x, self.y, self.Lz, top_field)
+        xz_side = construct_surface_dict(self.x, self.Ly, self.z, right_field)
+        yz_side = construct_surface_dict(self.Lx, self.y, self.z, left_field)
+        
+        
+        cmap = matplotlib.cm.get_cmap(self.cmap)
+        vmin, vmax = self._get_minmax(left_field)
+        self.current_vmin, self.current_vmax = vmin, vmax
+        norm = matplotlib.colors.Normalize(vmin=vmin, vmax=vmax)
+        for d in [xy_side, xz_side, yz_side]:
+            x = d['x']
+            y = d['y']
+            z = d['z']
+            sfc = cmap(norm(d['surfacecolor']))
+            surf = ax.plot_surface(x, y, z, facecolors=sfc, cstride=1, rstride=1, linewidth=0, antialiased=False, shade=False)
+            ax.plot_wireframe(x, y, z, ccount=1, rcount=1, linewidth=1, color='black')
+        
+        
+
+        ax.view_init(azim=25, elev=10)
+
+
```

Comment:
delete white space and commented lines; add keyword args in __init__ to specify azim and elev, use these values as the defaults.